Document the container start/stop commands (#206)
anbox-cloud/discourse-docs: exp/containers.md

## Container life cycle

When you [create a container](https://discourse.ubuntu.com/t/launch-a-container/24327) by either launching or initialising an application or an image, it executes the following steps in order:

> 1. Schedule a container on LXD node.

Reviewer: The "it" in "it executes the following steps in order" refers to the container - that doesn't fit with the first step here, since the container probably doesn't schedule itself. ;)
So either change the "it" to "LXD" and make it clearer in the following steps which happen inside the container and which outside, or move the first step into the introduction. (For example: "... by either launching or initialising an application or an image, LXD schedules the container on a LXD node. The container then executes ..."

Author: Changing the "it" to "LXD" doesn't sound right to me considering the context. E.g. Execute the hook, it's performed by the container itself rather than the LXD. I'd vote to move the first step into the introduction as that sounds more reasonable and natural

1. Configure the network interface and gateway.
1. Only raw containers: Install addons that are specified with `--addons`.
1. Expose services that are specified with `--service` or through the application manifest.
1. Execute the `pre-start` hook provided by the installed addons.
1. Launch the Android container.
1. Execute the `post-start` hook provided by the installed addons.

![Container start|584x646](TBD)

The whole launch process is successful only if all of the above steps succeed.

If anything goes wrong during the container launch process, the status of the container changes to the `error` status. You can [view the available logs](https://discourse.ubuntu.com/t/view-the-container-logs/24329) from the container for diagnosing the root cause of the problem.

> When a container is stopped, because one of the following scenarios occurs:
> 
> - you stopped it.
> - you deleted it.
> - an error occurred.
> 
> it executes the following steps in order:

Reviewer: We shouldn't have a list like this in the middle of a sentence.
How about:
```suggestion
Containers can be stopped because of the following scenarios:

- You stopped it.
- You deleted it.
- An error occurred.

When a container is stopped, it executes the following steps in order:
```
